src/drafter.h

```diff
@@ -54,6 +54,8 @@ typedef enum
  */
 typedef struct {
     bool requireBlueprintName;
+    bool generateMessageBody;
+    bool generateMessageBodySchema;
```

Comment:
Every time we add an item to the options we are breaking the ABI. I wonder if it makes sense to redesign this in the future so we can introduce changes without breaking ABI stability?

Perhaps the internals of the structure can be hidden from the public interface are we can provide functions to set the values?

```c
drafter_parse_options *drafter_parse_options_alloc(void) {
    // FIXME - set all option defaults
    return (drafter_parse_options *)malloc(sizeof(drafter_parse_options));
}

void drafter_parse_options_dealloc(drafter_parse_options * options) {
    free(options);
}

drafter_parse_options *drafter_parse_options_set_generate_message_body(drafter_parse_options * options, bool generate_message_body) {
    options->generateMessageBody = generate_message_body;
}
```

@tjanc what do you think?

Reply:
Larger example: https://github.com/kylef-archive/geoffrey/blob/c197c9ed6ed550aec74e53ef2842149eb3520659/src/main.c#L7 // https://github.com/kylef-archive/geoffrey/blob/c197c9ed6ed550aec74e53ef2842149eb3520659/examples/echo.c#L41

Reply:
Discussed in sprint planning, here is the output:

- We'll create an interface such as https://github.com/apiaryio/drafter/pull/746/files#r324681068 which allows us to add and change options without breaking ABI stability.

Reply:
@pksunkara Have I missed something? Is this implemented as discussed?